<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quitBtn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="messageBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>MESSAGE!!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="infoBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>INFO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="northBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>NORTH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="west">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>WEST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eastBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>EAST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="southBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>SOUTH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="teleportBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>TELEPORT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>101</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>ELements:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mapImg">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>441</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap>img/map.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="collectBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>COLLECT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
